src/actions/index.js

```diff
@@ -1,7 +1,20 @@
 // we'll need axios
-
+import axios from 'axios';
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
+export const CHARS_FETCHING = 'CHARS_FETCHING';
+export const CHARS_SUCCESS  = 'CHARS_SUCCESS';
+export const CHARS_FAILURE = 'CHARS_FAILURE';
+
+export const getChars = () => dispatch => {
+    dispatch ({type: CHARS_FETCHING });
+    axios
+        .get(`https://swapi.co/api/people/`)
+        .then(res =>
+            dispatch({type: CHARS_SUCCESS, payload: res.data.results}))
```

Comment:
An alternative way of handling this is by destructuring the arguments as follows:
```
.then(({ data }) => {
      dispatch({type: FETCHING_CHARACTERS_SUCCESS, payload: data.results}))
```
Which basically just means that you don't need to worry about the actual `res` response object.